homeassistant/components/synology_dsm/diagnostics.py

```diff
@@ -64,21 +67,26 @@ async def async_get_config_entry_diagnostics(
 
         volume: dict
         for volume in syno_api.storage.volumes:
-            diag_data["storage"]["volumes"][volume["id"]] = {  # type: ignore[index]
+            diag_data["storage"]["volumes"][volume["id"]] = {
                 "name": volume["fs_type"],
                 "size": volume["size"],
             }
 
     if syno_api.surveillance_station is not None:
         camera: SynoCamera
         for camera in syno_api.surveillance_station.get_all_cameras():
-            diag_data["surveillance_station"]["cameras"][camera.id] = {  # type: ignore[index]
+            diag_data["surveillance_station"]["cameras"][camera.id] = {
                 "name": camera.name,
                 "is_enabled": camera.is_enabled,
                 "is_motion_detection_enabled": camera.is_motion_detection_enabled,
                 "model": camera.model,
                 "resolution": camera.resolution,
             }
+            camera_data = await camera_diagnostics.async_get_config_entry_diagnostics(
+                hass, entry
+            )
```

Comment:
Did you mean to put this inside the for loop?

Reply:
```
    "surveillance_station": {
      "cameras": {
        "1": {
          "name": "Front Door",
          "is_enabled": true,
          "is_motion_detection_enabled": true,
          "model": "Generic_ONVIF",
          "resolution": "1280x720"
        }
      },
      "camera_diagnostics": {
        "camera.alexanderalpha_front_door": {}
      }
```

This is what I get from it

Reply:
> Did you mean to put this inside the for loop?

ah, you are right ... one indention to deep 🙈 

> This is what I get from it

this is correct, since camera diagnostics are just stream diagnostics, which needs to have a stream started first